tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+    "include": [
+        "src/**/*.js",
+        "index.js"
+    ],
+    "compilerOptions": {
+        "target": "es2019",
```

Comment:
Just a suggestion, but I think use the latest compiler version? That is completely up to you